### ISSUE TITLE
Test and formatter not running with Elixir v1.16.0-rc.0

### ISSUE THREAD

**Issue by ypconstante** [karolsluszniak/ex_check#41]
I'm testing Elixir v1.16, and ex_check is not running test and formatter steps, saying it's missing files

.check.exs
```
[
  fix: true,
  retry: false,
  tools: [
    {:doctor, false},
    {:ex_doc, false},
    {:mix_audit, false},
  ]
]

```

`mix check` output
```
=> finished in 0:38

 ✓ compiler success in 0:16
 ✓ credo success in 0:22
 ✓ dialyzer success in 0:18
 ✓ sobelow success in 0:11
 ✓ unused_deps fix success in 0:00
   ex_unit skipped due to missing file ../test
   formatter skipped due to missing file ../.formatter.exs

```

**Comment by karolsluszniak** [comment]
Any apparent root case? I guess I'll get to it myself upon update to 1.16 but that won't be before final & I have too much on my plate ATM so if anyone could diagnose & fix this while it's in RC it'd be awesome!

**Comment by ypconstante** [comment]
This will be fixed on Elixir 1.16.2 https://github.com/elixir-lang/elixir/issues/13310
